dapr.code-workspace

```diff
@@ -14,6 +14,9 @@
 		},
 		{
 			"path": "samples/python-azurefunction"
+		},
+		{
+			"path": "test/DaprExtensionTests"
```

Comment:
I'm not super sure why there's a workspace config here - using it makes it harder to work on the .NET code 🤷 since you can't resolve references across workspace folders. It's better to just load up the solution directly.

Reply:
There's no particular reason to have this. I think it can be removed.

Reply:
@cgillum - do you mean remove the file wholesale or undo the edit?

Reply:
I meant the whole file. This was added by Jeff before I started working on this project and I never found it particularly useful.